mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -223,7 +223,7 @@ public void user2ShouldBeAbleToCreateNestedFoldersWithoutTheirParents() throws B
 
```

Comment:
I mean you probably lack a test where the mailbox you try to put annotation on doesn't exists.